tests/nnstreamer_filter_mxnet/download_and_run.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Nope. codes in main nnstreamer.git should use LGPL 2.1

@minty99 For test scripts, being LGPL 2.1 doesn't incur problems. Please apply LGPL 2.1 instead of Apache-2.0